test/recipes/70-test_sslvertol.t

```diff
@@ -34,33 +34,69 @@ my $proxy = TLSProxy::Proxy->new(
     (!$ENV{HARNESS_ACTIVE} || $ENV{HARNESS_VERBOSE})
 );
 
+my @available_tls_versions = ();
+foreach (available_protocols("tls")) {
+    unless (disabled($_)) {
+        note("Checking enabled protocol $_");
+        m|^([a-z]+)(\d)(_\d)?|;
+        my $versionname;
+        if (defined $3) {
+            $versionname = 'TLSProxy::Record::VERS_'.uc($1).'_'.$2.$3;
+            note("'$1', '$2', '$3' => $versionname");
+        } else {
+            $versionname = 'TLSProxy::Record::VERS_'.uc($1).'_'.$2.'_0';
+            note("'$1', '$2' => $versionname");
+        }
+        push @available_tls_versions, eval $versionname;
+    }
+}
+note("TLS versions we can expect: ", join(", ", @available_tls_versions));
+
 #This file does tests without the supported_versions extension.
 #See 70-test_sslversions.t for tests with supported versions.
-#Test 1: Asking for TLS1.4 should pass and negotiate TLS1.2
+#Test 1: Asking for TLS1.4 should pass and negotiate the maximum
+#available TLS version according to configuration
 my $client_version = TLSProxy::Record::VERS_TLS_1_4;
-$proxy->clientflags("-no_tls1_3");
```

Comment:
We are trying to test what happens without the supported_versions extension which this flag gives us - so this needs to go back in.